### ISSUE TITLE
ImageNet pre-trained model code and hyper-parameters

### ISSUE THREAD

**Issue by Jobanan** [pytorch/vision#2469]
Hi,

Is the code used to trained the torchvision models (especially Resnet) on ImageNet available ? What are the hyper-parameters used  ? Did you use specific methods (dropout, weight decay, specific augmentation such as cutout...etc) during training ? 

Thank you very much

**Comment by pmeier** [comment]
Hi @Jobanan, you can find the training scripts for all pre-trained models here:

https://github.com/pytorch/vision/tree/master/references

The hyper parameters are displaying in the [README](https://github.com/pytorch/vision/blob/master/references/classification/README.md), but for everything else you have to dig into the actual [training script](https://github.com/pytorch/vision/blob/master/references/classification/train.py). For example, we used the following image transformation during training:

https://github.com/pytorch/vision/blob/0344603ece20ba5e4086aa8c49b2cce0829aefe6/references/classification/train.py#L96-L103

Let me know if you have further questions.
